### DISCUSSION TITLE
Exit Script Mode

### DISCUSSION THREAD

**Discussion by Pixydome** [Stellarium/stellarium#3113] [General]
Hello Everyone,

I have a question about Scripts Mode.

In effect, before run another script, i have an configuration graphic on my planetarium.

I run the script and for any reason i want to stop the script. 

If the script finnish correctly i return to my good configuration (date, hours, screen option, graphical interface)

If i stop the script manualy i cant return to my graphical initial position like just before i start the script.

I can't retrieve this configuration i must exit Stellarium for retrieve my inital position when i have configure on the Stellarium option.

There is possible to retrieve the good configuration when i exit manualy another scripts ?

Thanks in advance for your feedback.

Brest regards,

Arnaud VANDOMME

**Comment by gzotti** [comment]
No, there is no "Save current state, begin script, if script was cancelled restore state". Scripts change state just like user actions.

**Comment by Pixydome** [comment]
Hello,

Thanks a lot for your feedback.

I will submit my idea for add feature.

Have a great day

Best regards,

Arnaud VANDOMME
